src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -197,6 +197,7 @@ default boolean onBlockStartBreak(ItemStack itemstack, BlockPos pos, Player play
      * @param count  The amount of time in tick the item has been used for
      *               continuously
      */
+    @Deprecated(since = "1.20")
```

Comment:
I would add to the javadoc `@deprecated use {@link Item#onUseTick(Level,LivingEntity,ItemStack,int)}` so its clear what the replacement is.

Same below for the item stack method.

Reply:
Done.